gradle/spec/dependabot/gradle/file_updater_spec.rb

```diff
@@ -124,6 +125,68 @@
 
       its(:content) { is_expected.to include "version: '4.2.0'" }
 
+      context "with a lockfile" do
+        let(:buildfile_folder) { "buildfiles/lockfile/app" }
+        let(:buildfile_fixture_name) { "build.gradle.kts" }
+        let(:buildfile) do
+          Dependabot::DependencyFile.new(
+            name: "app/build.gradle.kts",
+            directory: "app/",
+            content: fixture(buildfile_folder, buildfile_fixture_name)
+          )
+        end
+        let(:lockfile) do
+          Dependabot::DependencyFile.new(
+            name: "app/gradle.lockfile",
+            directory: "app/",
+            content: fixture(buildfile_folder, "gradle.lockfile")
+          )
+        end
+        let(:dependency_files) do
+          [
+            buildfile,
+            lockfile,
+            Dependabot::DependencyFile.new(
+              name: "settings.gradle.kts",
+              directory: "/",
+              content: fixture(buildfile_folder, "..", "settings.gradle.kts")
+            )
+          ]
+        end
+
+        let(:dependencies) do
+          [
+            Dependabot::Dependency.new(
+              name: "com.google.code.gson:gson",
+              version: "2.8.9",
+              package_manager: "gradle",
+              requirements: [{
+                file: "app/build.gradle.kts",
+                requirement: "2.8.9",
+                groups: [],
+                source: nil,
+                metadata: nil
+              }],
+              previous_requirements: [{
+                file: "app/build.gradle.kts",
+                requirement: "2.8.8",
+                groups: [],
+                source: nil,
+                metadata: nil
+              }]
+            )
+          ]
+        end
+
+        it "updates build.gradle.kts and gradle.lockfile" do
+          expect(updated_files.map(&:name)).to eq(["app/build.gradle.kts", "app/gradle.lockfile"])
+          expect(updated_files.first.content)
+            .to include('implementation("com.google.code.gson:gson:2.8.9")')
+          expect(updated_files.last.content)
+            .to include("com.google.code.gson:gson:2.8.9=runtimeClasspath")
```

Comment:
You may want to assert the entire line here. For example, in this instance, we're missing `,testCompileClasspath,testRuntimeClasspath`

Reply:
#ItsComplicated. We're running into what in my limited knowledge looks like https://github.com/gradle/gradle/issues/21396, such that we have both of these lines in the lockfile:
```
com.google.code.gson:gson:2.8.8=testRuntimeClasspath
com.google.code.gson:gson:2.8.9=compileClasspath,runtimeClasspath,testCompileClasspath
```

I can still assert the complete line, it just won't include `testRuntimeClasspath`.